core/src/test/java/com/linecorp/armeria/server/HttpServerKeepAliveHandlerTest.java

```diff
@@ -150,6 +155,10 @@ void closeByClientIdleTimeout(SessionProtocol protocol) throws InterruptedExcept
         await().untilAtomic(counter, Matchers.is(0));
         final long elapsed = stopwatch.stop().elapsed(TimeUnit.MILLISECONDS);
         assertThat(elapsed).isBetween(clientIdleTimeout, serverIdleTimeout - 1000);
+        assertThat(MoreMeters.measureAll(meterRegistry))
+                .hasEntrySatisfying("armeria.client.connections.lifetime#total", value -> {
+                    assertThat(value).isCloseTo(TimeUnit.MILLISECONDS.toSeconds(elapsed), withinPercentage(25));
```

Comment:
Shouldn't we make the `value` to milliseconds because `TimeUnit.MILLISECONDS.toSeconds(elapsed)` will shrink the `elapsed`?

Reply:
Oops...

Reply:
In Micrometer (and Prometheus), all duration values are represented as seconds.

Reply:
It just needs to be double so that sub-second part is preserved.